functions/src/wort/core.clj

```diff
@@ -2,6 +2,13 @@
   (:gen-class
    :methods [^:static [handler [String] String]]))
 
+(require '[clojure.string :as str])
+
 (defn -handler
   [s]
   (str "Wort " s " ツ"))
+
+(defn get_key_as_list
```

Comment:
Tests? :P

You don't have to add them, but it would be cool to see a first test or two and understand the command to run them (then we can add it to our build).